tests/test_user.py

```diff
@@ -202,7 +202,7 @@ def test_mention_markdown(self, user):
         expected = u'[{}](tg://user?id={})'
 
         assert user.mention_markdown() == expected.format(user.full_name, user.id)
-        assert user.mention_markdown('the_name*\u2022') == expected.format('the\_name\*\u2022',
+        assert user.mention_markdown('the_name*\u2022') == expected.format('the\\_name\\*\u2022',
```

Comment:
Why is this change?

Reply:
Same reason as above - the evaluated string literal is the same though this is uses a proper escape sequence